src/Raven.Server/Documents/TimeSeries/TimeSeriesStats.cs

```diff
@@ -210,6 +210,13 @@ public long UpdateStats(DocumentsOperationContext context, TimeSeriesSliceHolder
 
             void HandleLiveSegment()
             {
+                if (segment.NumberOfEntries == 1 && start > end)
```

Comment:
Under what scenario do you have `not (start > end) ` ?

Reply:
`start > end` only when table doesn't exist (`start` is initialized to `DateTime.MaxValue` and `end` to `DateTime.MinValue`)